libs/design/scss/theming/create-theme/_index.scss

```diff
@@ -0,0 +1 @@
+@forward 'create-theme' show daff-create-theme;
```

Comment:
@sunray4 note @xelaint 's change here. In short, this prevents exposing every variable / function / mixin in the file to the public_api. @xelaint and I haven't done a good job with this in the past, but this seems like a good opportunity to start!